toast-tk-runtime/src/main/java/io/toast/tk/runtime/AbstractTestPlanRunner.java

```diff
@@ -119,7 +119,7 @@ private void updateTestPlanFromPreviousRun(final ITestPlan testPlan, final TestP
 		}
 	}
 
-	public void execute(final ITestPlan project, final boolean presetRepoFromWebApp) throws Exception {
+	public void execute(final ITestPlan testPlan, final boolean presetRepoFromWebApp) throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00112)